sql/chunk_constraint.sql

```diff
@@ -96,6 +117,19 @@ BEGIN
     AND cc.chunk_id = drop_chunk_constraint.chunk_id
     RETURNING * INTO STRICT chunk_constraint_row;
 
+    SELECT * INTO STRICT constraint_row
+    FROM pg_constraint con
+    WHERE con.conrelid = format('%I.%I', chunk_row.schema_name, chunk_row.table_name)::regclass
+    AND con.conname = constraint_name;
+
+    IF constraint_row.conindid IS NOT NULL THEN
+        DELETE FROM _timescaledb_catalog.chunk_index ci
```

Comment:
Again, we already have a function for this called `chunk_index_delete()`. I strongly think we shouldn't split this functionality across two places and two code bases, for reasons I've expanded on in related comments.

Reply:
Did you miss this?

Reply:
Yes. Fixed now